### DISCUSSION TITLE
what is the best way to migrate from version 2.0.0-preview021 to latest ?

### DISCUSSION THREAD

**Discussion by justicejoe** [prisma/prisma#4057] [Q&A] [answered]
I developed this project at the start of the year, then take a break to another project until now. So my current version is so outdated right now. Is there a guide to migrate to the latest one? I just come to know that prisma2 is changed to prisma-cli.

**Comment by ryands17** [comment, marked as answer]
@justicejoe There were changes made specifically that you would need to check [here](https://github.com/prisma/prisma/releases/tag/2.0.0-preview022). Currently there's `@prisma/client` and `@prisma/cli` instead of Prisma 2.

The rest of your relations can be validated by looking at this [doc](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-schema/relations#overview) on how relations are handled in the latest version.

The rest would all the the same in the way you query for stuff and extra functionality has been added since then as Prisma Client has become stable.

**Reply by justicejoe** [reply]
Ok Sorry, I forgot to mention that this project of mine has also worked with nexus-prisma and nexus. Since nexus-prisma is changed to nexus-plugin-prisma and nexus is changed to @nexus/schema. Are they still working with the latest version of prisma? 
When I try to migrate all of these to a new version, all of the basic t.crud disappear from the graphql playground. Only custom t.field remain.

**Reply by ryands17** [reply]
@justicejoe yes it wil work fine. For `t.crud` you would need to explicitly enable this.

So it would look like:

```ts
import { nexusSchemaPrisma } from 'nexus-plugin-prisma/schema

const nexusPrisma = nexusSchemaPrisma({
  experimentalCRUD: true,
  prismaClient: (ctx: Context) => ctx.prisma,
})
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
